### PR TITLE
AC: allow to process result without resize meta

### PR DESCRIPTION
**Pull request by eaidova**
with enabling preprocessing throw ie, some preprocessing metadata missed.
adapters which use such info are updated to work without it (by default scaling coefficients calculated like for case with single resize to input resolution - ie resize behaviour. if coeffs provided, then we use them (they can be different from default, for example, if we use aspect ratio preserving resize + padding to input size)

### CODE REVIEW

File: tools/accuracy_checker/accuracy_checker/adapters/detection.py [hunk 1]
```diff
@@ -429,12 +429,26 @@ def configure(self):
         self.rois_out = self.get_value_from_config('rois_out')
 
     def process(self, raw, identifiers=None, frame_meta=None):
+        assert len(identifiers) == 1, '{} adapter support only batch size 1'.format(self.__provider__)
```
[the remark this answers is not in the document]
Comment (eaidova) [reply tools/accuracy_checker/accuracy_checker/adapters/detection.py:433]: default values were rudiment from pipelined mode (which was removed), I updated process method signature

File: tools/accuracy_checker/accuracy_checker/adapters/detection.py [hunk 2]
```diff
@@ -429,12 +429,26 @@ def configure(self):
...
-        y_scale = frame_meta[0]['scale_y']
+        meta = frame_meta[0]
+        if 'scale_x' in meta:
+            x_scale = meta['scale_x']
+            y_scale = meta['scale_y']
+        else:
+            original_image_size = meta['image_size'][:2]
+            image_input = [shape for shape in meta['input_shape'].values() if len(shape) == 4]
+            assert image_input, "image input not found"
+            image_input = image_input[0]
```
[the remark this answers is not in the document]
Comment (eaidova) [reply tools/accuracy_checker/accuracy_checker/adapters/detection.py:447]: sure, added

File: tools/accuracy_checker/accuracy_checker/adapters/retinaface.py [hunk 3]
```diff
@@ -238,3 +236,20 @@ def landmark_pred(boxes, landmark_deltas):
...
         return pred
+
+    @staticmethod
+    def get_scale(meta):
+        if 'scale_x' in meta:
+            return meta['scale_x'], meta['scale_y']
+        original_image_size = meta['image_size'][:2]
+        image_input = [shape for shape in meta['input_shape'].values() if len(shape) == 4]
+        assert image_input, "image input not found"
+        image_input = image_input[0]
```
Comment (IRDonch) [inline tools/accuracy_checker/accuracy_checker/adapters/retinaface.py:247]: Need to verify ythat there's only one image here, as well.

Comment (eaidova) [reply tools/accuracy_checker/accuracy_checker/adapters/retinaface.py:248]: added

**Pull request merged**